app/views/orgs/assignment_repos/_assignment_repo.html.erb

```diff
@@ -1,4 +1,4 @@
-<include-fragment src="<%= url %>" class="flex-table assignment-repo-list-item">
```

Comment:
The `assignment-repo-list-item` has some extra styles for the spinner and alert inside. This is causing the alert and spinner to show up at the same time:

![Screen Shot 2019-06-06 at 12 42 42 AM](https://user-images.githubusercontent.com/7866060/58983926-74555f00-87f5-11e9-942f-ce1c06e88adf.png)

Reply:
Might want to include the class on all of these just for the custom styling for the error states.

Reply:
Just fixed the styling on these rows by adding primer-styles + using the custom class for the error handling